baseplate/diagnostics/tracing/publisher.py

```diff
@@ -0,0 +1,191 @@
+import argparse
+import logging
+import time
+
+import requests
+
+from . import MAX_SPAN_SIZE, MAX_QUEUE_SIZE
+from baseplate import config, metrics_client_from_config
+from baseplate._compat import configparser
+from baseplate.events.publisher import BatchFull, V1Batch, TimeLimitedBatch
```

Comment:
we may want to move these to a more common location (`baseplate._utils`?) `V1Batch` is a pretty awkward name here, so it may be good to rename that to `RawJSONBatch` or something and then subclass it to `TraceBatch` and `V1Batch` in the respective clients?

Reply:
Agreed. I thought about that but didn't want to introduce a refactor into these changes. If you think that's needed to get the rest of this in, I'll start on it.